app/services/idv/session.rb

```diff
@@ -0,0 +1,115 @@
+module Idv
+  class Session
+    delegate :questions, to: :resolution
+
+    def initialize(user_session, current_user)
+      @user_session = user_session
+      @current_user = current_user
+      @user_session[:idv] ||= {}
+    end
+
+    def attempts=(num)
+      current_user.update!(idv_attempts: num)
+    end
```

Comment:
How about moving this and `flag_user_attempt` to a class called `UpdateAttemptsCount` with methods `reset` (which sets the count to zero), and `increment`? I don't think we want to have a public `attempts=` method, because that would allow setting the attempts to any arbitrary value, and signal that it's OK to do so, but the only values we allow are either zero when resetting the count, or incrementing the current count by 1.